src/js/components/aboutTheData/AboutTheDataModal.jsx

```diff
@@ -44,7 +44,7 @@ const AboutTheDataModal = ({
             <div className={`usa-dt-modal about-the-data-modal ${className}`}>
                 <div className="usa-dt-modal__header">
                     <div className="about-the-data-modal__header-data">
-                        <div className="about-the-data-modal__agency-name">{agencyData.agencyName}</div>
+                        <div className="about-the-data-modal__agency-name">{agencyData.agencyName ? agencyData.agencyName.toUpperCase() : ''}</div>
```

Comment:
Would it make more sense to do this formatting in the model?

Reply:
I dont think we should have the casing set in the modal, because we would have to set the casing to title case in the table.

Reply:
Ah I see, it differs between the modals and the table